### PR TITLE
Detailed organization bootstrapping guide

### PR DESCRIPTION
**Pull request by cblkwell**
So, I went back and finished something I got started on a long time ago -- some of this was already sort of included in the basic page on AWS organizations, but after the questions from Chris and the other MilMove folks about how organizations work, I decided to revive this idea and I wrote a thorough guide with lots of code examples to hopefully illustrate these concepts better. Looking for review on multiple levels with this one.

### CODE REVIEW

File: infra/aws/org-bootstrap.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+bootstrap your `id` account. Once you've done that, you can set up the
+users and groups that team members will use to access all other AWS
+resources. Truss has made two modules which will help you get this done,
+[`iam-cross-acct-src`](https://registry.terraform.io/modules/trussworks/iam-cross-acct-src/aws) and
+[`iam-cross-acct-dest`](https://registry.terraform.io/modules/trussworks/iam-cross-acct-dest/aws).
+
+In order to do this, in your `id` account, define your users and then
+use the module like so:
+
+```hcl
```
Comment (dynamike) [inline infra/aws/org-bootstrap.md:259]: Same suggestion as in org-root
```hcl
locals {
  infra_users = [
    "myuser",
  ]
}

resource "aws_iam_user" "infra" {
  for_each      = toset(local.infra_users)
  name          = each.value
  force_destroy = true
  tags = {
    Automation = "Terraform"
  }
}

resource "aws_iam_group_policy_attachment" "infra" {
  group      = aws_iam_group.infra.name
  policy_arn = "arn:aws:iam::aws:policy/PowerUserAccess"
}

resource "aws_iam_group_membership" "infra" {
  name = "infra"
  users = values(aws_iam_user.infra)[*].name
  group = aws_iam_group.infra.name
}

module "infra_group" {
  source  = "trussworks/iam-user-group/aws"
  version = "1.0.2"
  user_list     = values(aws_iam_user.infra)[*].name
  allowed_roles = [module.infra_group_role.arn]
  group_name    = "infra"
}
```

File: infra/aws/org-bootstrap.md [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+[profile spacecats-infra]
+source_profile = spacecats-id
+role_arn = arn:aws:iam::<spacecats-infra account number>:role/infra
+region = us-west-2
+output = json
+# If you have MFA turned on, you will also need to add the mfa_serial
+# line from the source_profile.
+```
+
+## Next Steps
```
Comment (dynamike) [inline infra/aws/org-bootstrap.md:354]: And additional next step could be to revisit the org-scp module README to enable SCPs like denying access to the root user in every account.

File: infra/aws/org-bootstrap.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+If you use a shared resource, like an S3 bucket, you end up having to
+do more detailed permission management to keep accounts from stepping on
+each other -- this essentially eliminates the advantage of the
+organization in the first place.
+
+For most things, creating a separate resource for each account is likely
+the better play -- for things like S3 buckets, having two S3 buckets with
+half as much stuff as a single larger S3 bucket ends up essentially being
+the same cost. Even for things where the added cost is marginal, like
+having multiple ECR repositories, the added peace of mind keeping your
```
Comment (dynamike) [inline infra/aws/org-bootstrap.md:384]: I'm not sure multiple ECRs is the right example for exemplifying why it makes sense to isolate resources on a per account basis. While the additional costs may not be a big deal, the complexity and time this kind of design adds to a delivery pipeline is a negative tradeoff. I would personally lean towards putting the ECR in the infra account.

How about using examples like separating VPCs or ECS services to each account instead?

File: infra/aws/org-bootstrap.md [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+
+# Organization Accounts
+
+resource "aws_organizations_account" "spacecats_id" {
+  name                       = "spacecats-id"
+  email                      = "spacecats-infra+aws-org-id@example.com"
+  parent_id                  = aws_organizations_organizational_unit.spacecats.id
+
+  # Allow IAM user access to billing for this account so that we can let
+  # project/delivery managers access billing info without a root account.
```
[the remark this answers is not in the document]
Comment (cblkwell) [reply infra/aws/org-bootstrap.md:189]: Womp womp. Well, it's still good advice for folks in the future.

File: infra/aws/org-bootstrap.md (hunk 2 above)
Comment (cblkwell) [reply infra/aws/org-bootstrap.md:354]: Good call. I think there's some SCPs we can take from SABER and add to the org-scp module that might be useful too...I'll make a ticket for that.

File: infra/aws/org-bootstrap.md [hunk 5]
```diff
@@ -0,0 +1,394 @@
...
+Once those are set up, you can begin the rest of the work. Your first
+action should be to create administrative IAM users that will *only* be
+used to manage the master account and bootstrap the subordinate accounts.
+These should not be used for anything else. Suffix the user name with
+`.org-root` to make it clear what these are for. They will be the only
+accounts that can manage the master account.
+
+Example Terraform code (suggest using a `users.tf` file):
+
+```hcl
```
[the remark this answers is not in the document]
Comment (cblkwell) [reply infra/aws/org-bootstrap.md:84]: Went through and revised this to use the Truss `iam-user-group` module too, just for funsies.

File: infra/aws/org-bootstrap.md (hunk 1 above)
Comment (cblkwell) [reply infra/aws/org-bootstrap.md:262]: Done.

File: infra/aws/org-bootstrap.md (hunk 2 above)
Comment (cblkwell) [reply infra/aws/org-bootstrap.md:363]: Added some language about this.

File: infra/aws/org-bootstrap.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (dynamike) [reply infra/aws/org-bootstrap.md:84]: sounds like a good time

**Pull request merged**